scripts/azure_deployment/arm_aci.py

```diff
@@ -88,73 +89,130 @@ def make_aci_deployment(parser: ArgumentParser) -> Deployment:
         type=lambda comma_sep_str: comma_sep_str.split(","),
     )
 
+    parser.add_argument(
+        "--aci-deploy-external-executer",
+        help="Deploy external executer container if true. Default=False",
+        default=False,
+        type=bool,
+    )
```

Comment:
With the current code, `arm_template.pyt deploy` deploys attestation container when this flag is true because it's a sidecar of external executer.
Even it's true only attestation container is deployed for now.